### PR TITLE
How Grain Works

### PR DESCRIPTION
**Pull request by Bex2594**
The GitHub reorg deleted my version of the document, as well as Joie's revision. This is the version I originally authored for the beta docs

### CODE REVIEW

**Comment by Bex2594** [comment]
@KuraFire  I published a new branch to pull to master, but I'm unsure how GitHub would know which document I'm trying to pull to. Dandelion did a GitHub reorg recently and my original doc was deleted, so I published a new branch (Bex2594-How-Grain-Works). Is it somehow connected to the doc 'grain.md' in the beta folder?

File: docs/beta/grain.md [hunk 1]
```diff
@@ -1,85 +1,50 @@
...
+### How do I get Grain?
 
-Grain is created during "harvests." In a harvest, a project mints new grain and
-distributes it to contributors based on their cred scores. How the grain is
-harvested and distributed is up to the project. For example, a project could
-mint a fixed amount of grain every week and distribute it to contributors based
-on their recent contributions, or it could have a target "grain-per-cred"
-payout ratio, and harvest to ensure that all contributors receive at least that
-amount.
+**CHANGE MONEY BAG EMOJI TO GRAIN TOKEN EMOJI IN LAST SLIDE: https://docs.google.com/presentation/d/1CIHI4xcDCiTgD2ACMG8Jnbv843Gq45V3SHKex915WY0/edit#slide=id.g8703f78739_0_409**
```
Comment (Bex2594) [inline docs/beta/grain.md:15]: *[//]: # (CHANGE MONEY BAG EMOJI TO GRAIN TOKEN EMOJI IN LAST SLIDE: https://docs.google.com/presentation/d/1CIHI4xcDCiTgD2ACMG8Jnbv843Gq45V3SHKex915WY0/edit#slide=id.g8703f78739_0_409)*

Use Dandelion's new comment syntax

File: docs/beta/grain.md [hunk 2]
```diff
@@ -1,85 +1,50 @@
...
+**SECTION COULD BE MORE DIRECT**
```
Comment (Bex2594) [inline docs/beta/grain.md:16]: *[//]: # (SECTION COULD BE MORE DIRECT)*

Use Dandelion's new comment syntax

File: docs/beta/grain.md [hunk 3]
```diff
@@ -1,85 +1,50 @@
...
+A concept guide on Boosting will be coming soon.
 
-## SourceGrain
+# 🔑 Other key properties of Grain
 
-SourceCred is itself issuing and using grain. We call our grain 'SourceGrain'.
-SourceGrain isn't yet "officially live", but you can see our [preview
-balances].
+### Grain is transferrable and tradeable.
+Holders of Grain can freely send it to others, or exchange it. Within a project, Grain is fungible. If Grain has been put on a blockchain (e.g. as an ERC20 token), then it can be swapped or traded via protocols like Uniswap or 0x, just like any other ERC20. **MAKE MORE CLEAR**
```
Comment (Bex2594) [inline docs/beta/grain.md:46]: *[//]: # (MAKE MORE CLEAR)*

Use Dandelion's new comment syntax

**Comment by KuraFire** [comment]
> @KuraFire I published a new branch to pull to master, but I'm unsure how GitHub would know which document I'm trying to pull to. Dandelion did a GitHub reorg recently and my original doc was deleted, so I published a new branch (Bex2594-How-Grain-Works). Is it somehow connected to the doc 'grain.md' in the beta folder?

You don't really "pull to a document" (I know, it's confusing). GitHub will track internally if a file gets moved in a commit. Right now, the branch for this pull request (#143), that we're having this conversation in, is updating the file `docs/beta/grain.md` and that seems to be the right one at this time, so you're all set!  :)

**Pull request merged**